tests/EndToEndBuildTests/ProvidedTypes/ProvidedTypes.fs

```diff
@@ -48,9 +48,6 @@ namespace ProviderImplementation.ProvidedTypes
             let toObj x = match x with UNone -> null | USome x -> x
             let ofObj x = match x with null -> UNone | x -> USome x
 
-
-        let tryFindMulti k map = match Map.tryFind k map with Some res -> res | None -> [| |]
-
         let splitNameAt (nm:string) idx =
```

Comment:
not this.  This file came from the type provider sdk, so we shouldn't change it.  You might submit a change to the type provider sdk though.